bundles/http/Citrix/Zoom-Meetings/e40510b3-a760-43b4-ab16-56572dc6303b.js

```diff
@@ -303,20 +307,20 @@ async function incSyncMeetings(dataStore,client,context,latestSynchronizationTim
         if (next_page_token == "") {
             i++;
         }
-    } while (i < context.user.length)
+    } while (i < users.length)
     return meetingids
 }
 
 
 
 //Defining Actions Methods
-async function createMeeting({ dataStore, client, actionParameters }) {
+async function createMeeting({ dataStore, client, serviceClient, actionParameters }) {
```

Comment:
```suggestion
async function createMeeting({ dataStore, serviceClient, actionParameters }) {
```

There doesn't seem to be any usage of "client" in that function

Reply:
We had some scope issues , for that we were checking some workaround with `serviceClient` but that doesn't work. I forgot to revert back the changes. We now will be using the `client` only